NSObject+Rx.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = "NSObject+Rx"
-  s.version      = "1.2.1"
+  s.version      = "1.2.2"
```

Comment:
Since this is a [breaking change](http://semver.org), setting the version to a major change of `"2.0.0"` might be better. Thoughts?